### DISCUSSION TITLE
[IntelliSense] Preview color in tailwind.config.cjs

### DISCUSSION THREAD

**Discussion by leElvyn** [tailwindlabs/tailwindcss#8568] [Ideas]
It would be nice if colors could be previewed in the `tailwind.config.cjs`, the same way they are when actually used. That way you could get an overview of your colors, and you don't have to guess how it actually looks like.

Color preview in classes : 

![image](https://user-images.githubusercontent.com/45512739/172937502-e65758fc-589e-4a0e-b5eb-518ac1aff4fc.png)

tailwind cjs currently : 
![image](https://user-images.githubusercontent.com/45512739/172937713-6193b969-ec72-437b-b2b0-2558c20a1c6e.png)

**Comment by crswll** [comment]
You probably want an extension like this: https://marketplace.visualstudio.com/items?itemName=naumovs.color-highlight

<img width="291" alt="Screen Shot 2022-06-09 at 9 38 21 PM" src="https://user-images.githubusercontent.com/182222/172972888-198448fb-e495-437d-9d25-1e93485ee64b.png">
